BoardWebApp-EF/Models/User.cs

```diff
@@ -71,6 +72,54 @@ public static string ComputeSha256HashForString(string rawString)
                 return hashedString.ToString();
             } 
         }
-                
+
+
+        //  Member Type   - Id
+        //  Owner         - 1
+        //  Member        - 2
+        //  Scrum Master  - 3
+
```

Comment:
Try and extract the query code in a separate method that takes in the member type as parameters.